src/fdw/trigger.rs

```diff
@@ -222,6 +236,15 @@ fn duckdb_type_to_pg(column_name: &str, duckdb_type: &str) -> Result<String> {
         .replace("TIMESTAMP_NS", "TIMESTAMP")
         .replace("TIME WITH TIME ZONE", "TIME");
 
+    // DuckDB spatial types conversions
+    postgres_type = postgres_type
```

Comment:
Is this something we could map to the PostGIS types in Postgres, assuming the extension is installed?

Reply:
Yes, they are mapped to PostGIS.

Reply:
Will this work if a user install pg_analytics but doesn't install PostGIS? We should make sure that it's going to be compatible no matter what, as we can't guarantee PostGIS will be instaleld

Reply:
It would be a problem if the user doesn't install PostGIS. I will change it to the native PostgreSQL data type.

Reply:
Is it possible to have both? Can it, say "if PostGIS is installed, use PostGIS types, otherwise use Postgres types"? Or is this not possible?

Reply:
I believe we don't need to take additional steps. From my understanding, DuckDB-Spatial abstracts all geometric types as the `GEOMETRY` data type when running the `st_read` function, which is stored as a `BLOB`. Therefore, we can treat it the same way we handle `BLOB` data. Later, if users want to manipulate the geometry using PostGIS or other extensions, it should still be compatible.

Reply:
Wonderful, thank you! Could you please add this as a comment to the code for reference?